GenumKit/Stencil/Contexts.swift

```diff
@@ -66,7 +66,9 @@ extension AssetsCatalogParser {
     - `name`: `String`
     - `scenes`: `Array` (absent if empty)
        - `identifier`: `String`
-       - `class`: `String` (absent if generic UIViewController)
+       - `customClass`: `String` (absent if generic UIViewController)
+       - `isBaseViewController`: `Bool`
```

Comment:
maybe we should comment that a little more, to explain what we call "baseViewController" / "baseType", so that people wanting to implement their template understand what values those fields can have and what they mean